src/stories/initialData.js

```diff
@@ -5,4 +5,4 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-export const INITIAL_DATA = `<h2>CKEditor</h2><hr><h3>Basic styles</h3><p>Text can have <strong>bold</strong>, <i>italic</i>, and <u>underline</u> content. <code>Inline code</code> is also available. <a href="https://www.google.com">Links work as well.</a></p><ul><li>Lists</li><li>Are<ol><li>Nestable</li><li>And can change</li><li>Types</li></ol></li></ul><h3>Block elements</h3><blockquote><p>Block quotes can have inline content</p><p><strong>- Author name</strong></p></blockquote><pre><code class="language-plaintext">Preformatted code can go here</code></pre>`;
+export const INITIAL_DATA = `<h2>Heading 1</h2><hr><h3>Heading 2</h3><hr><h4>Heading 3</h4><hr><p>Text can have <strong>bold</strong>, <i>italic</i>, and <u>underline</u> content. <code>Inline code</code> is also available. <a href="https://www.google.com">Links work as well.</a></p><p style="margin-left:0px;">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><ul><li>Nested 1</li><li>Nested 2<ol><li>Another</li><li>Nested</li><li>List</li></ol></li></ul><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p><blockquote><p>Block quotes can have <i>inline <strong>content</strong></i></p><p><strong>- Author name</strong></p></blockquote><pre><code class="language-plaintext">Preformatted code can go hereIt can have extra spacing</code></pre>`;
```

Comment:
> * have you considered how the current HR treatment throws off vertical rhythm? A `<p>..<p>..<hr>..<p>..<p>` progression results in `16px..20px..20px..16px` of spacing. I'm going to keep asserting that 0 padding on the HR is the right amount (and provides a more general focus solution for "typearound" widgets – which would be very nice to deliver out of the box)

IMO although the 16/20px progression isn't consistent, I felt like it still feels [visually balanced](https://share.getcloudapp.com/mXu1Bq4R). I believe the current HR padding is 4px; I tested 2px with Austin at Allison's suggestion, and we weren't really able to select the HR with affordability to elements below/above the HR. If we went with 0, does this mean the touch affordance would become even worse? If that's the case I'd vote for keeping it as is.

> * list spacing is still tight compared to Garden's [component](https://garden.zendesk.com/components/lists) – has `::before`/`::after` pseudo been attempted to get the desired 2px?

I think the current line-height is the same as <p> text. We should try to match Garden if possible, although I do find the reduced spacing we have for nested lists visually disturbing (lol).

> * the lack of a sticky bottom bar is nearly unusable. Seems like designs are needed to spec out minimum, scrollable editing heights etc.

Agree with this. Can work this out with Austin for pattern specs at least.

> * are plans in place to fix the `rect` around the code block SVG?
This is on our radar but Austin mentioned it's out of scope for this first release.

Reply:
I've updated the initial data to have a more coherent nested list wording and code-block sizing.

+1 to everything @m-lai said. Having a `Paragraph -> List -> Paragraph` starts to look odd when the line-heights are handled differently. This is also compounded when there is wrapped text content within a list item.

Reply:
@austingreendev I agree. List spacing cannot be obtained by increasing the `line-height`. But have approaches been explored where `::before/::after` pseudo is used on the `<li>` in order to match the 2px div padding we see with the system? The system has already been considered with other components (i.e. Paragraph) in mind, and we should do everything possible to attempt to match that.

Reply:
@m-lai I'll ask and then stop pestering, I promise. Why does the HR need touch affordance? I just don't think it does and 0 padding is not only acceptable, but desirable for matching the intended HTML output.

Reply:
@jzempel I just had a sync with Austin and tried some stuff, the vertical rhythm is now 16px between `<p>...<hr>...<p>` 

>Re: Why does the HR need touch affordance?
The HR is focusable because it's not just for decoration. IMO if it has to be focusable, we might as well make it reasonably easier for users to click, if they wanted to (everything else inside the editor can be clicked on and modified). CKEditor considers it a widget for whatever reason. The typing cursor also takes on the box shadow height so it becomes a 1px blinking dot which looks weird (this adds to it but isn't the main reason why).